spec/schemas/ciag-induction/ciag-induction-created.yml

```diff
@@ -0,0 +1,55 @@
+name: ciag-induction.created
+title: CIAG Induction Created
+summary: A CIAG induction has been created for a Prisoner.
+contentType: application/json
+payload:
+  type: object
+  properties:
+    eventType:
+      type: string
+      example: "ciag-induction.created"
+    version:
+      type: integer
+      example: 1
+    detailUrl:
+      type: string
+      example: "https://learning-and-work-progress.hmpps.service.justice.gov.uk/plan/A0670DZ/view/overview"
+    occurredAt:
+      type: string
+      example: "2023-10-05T12:00:00.000Z"
```

Comment:
Is it worth adding `format: date-time` to this property? (`occurredAt`)

Reply:
Good point, done.